routes/socket/user-events.js

```diff
@@ -3075,6 +3075,35 @@ module.exports.handlePlayerReport = (passport, data) => {
 		comment: data.comment,
 		isActive: true
 	});
+
+	if (!/^(afk\/leaving game|abusive chat|cheating|gamethrowing|stalling|botting|bther)$/.exec(playerReport.reason)) {

```

Comment:
Think you want `other` instead of `bther` 😁

Reply:
Fixed